res/ps_border.fs.glsl

```diff
@@ -56,6 +56,59 @@ vec4 draw_dotted_edge() {
   return mix(white, circleColor, circleColor.a);
 }
 
+vec4 draw_double_edge(float pos, float len) {
+  // Devided border to 3 parts, draw color on first and third part,
+  // leave second part blank.
+  float one_third_len = len / 3.0;
+
+  // The original algorithm should be check if pos in the blank part.
+  // That is, checking (1/3 * len) < pos < (2/3 * len)
+  //
+  // The idea here is, offset pos by (1/3 * len) then mod it by len.
+  // And if this value is lesser than (2/3 * len) then we're in
+  // blank part. Thus, we can use only 1 check instead of 2.
+  float compared_pos = mod(pos + one_third_len, len);
```

Comment:
I change my implementation so it is easier to understand. And also calculations in local space. Would you check it again? Thanks very much.